api/api.go

```diff
@@ -91,7 +120,7 @@ func (s *BlockChainAPI) FeeHistory(
 // eth_gasPrice (returns the gas price)
 // GasPrice returns a suggestion for a gas price for legacy transactions.
 func (s *BlockChainAPI) GasPrice(ctx context.Context) (*hexutil.Big, error) {
-	return (*hexutil.Big)(big.NewInt(10102020506)), nil
+	return (*hexutil.Big)(big.NewInt(8049999872)), nil
```

Comment:
maybe use a constant for this.

Reply:
This is just a dummy value for now, but I have added a constant for it in https://github.com/onflow/flow-evm-gateway/pull/11/commits/42be5cdbb8ca4b9b948dbce13f7ab926f6298d9b